jenkins-build-pack-api/Jenkinsfile

```diff
@@ -727,6 +809,47 @@ def loadBuildModules(buildModuleUrl){
 		environmentDeploymentMetadata = load "environment-deployment-metadata-loader.groovy"
 		echo "Environment deployment data loader module loaded successfully."
 
+		apigeeModule = load "apigee-module.groovy"
+		apigeeModule.initialize(configLoader.APIGEE, events)
+		echo "Apigee module loaded successfully."
 	}
 }
 
+/**
+ * Query AWS for the ARN of a lambda
+ * 
+ * @param stackName Stack name used to query for the lamba ARN.
+ * @return object with the ARN for the lambda.
+ */
+def getLambdaARN (stackName) {
```

Comment:
There were few changes suggested to move this function to its own module. Please see https://github.com/tmobile/jazz/pull/441.